edxval/models.py

```diff
@@ -660,7 +660,7 @@ class Meta:
         choices=TranscriptProviderType.CHOICES,
     )
     # TODO remove exists in step 3 of renaming.
-    exists = models.BooleanField(default=False, help_text=u'Transcript credentials state')
+    exists = models.NullBooleanField(default=False, help_text=u'Transcript credentials state')
```

Comment:
If you aren't sure, I would ask you to check that this actually causes the column to be nullable, since for django fields in general null=false is typically the default.

Reply:
According to doc for BooleanField:  "Changed in Django 2.1:  In older versions, this field doesn’t permit null=True, so you have to use NullBooleanField instead. Using the latter is now discouraged as it’s likely to be deprecated in a future version of Django."   So null=True is the whole purpose of the NullBooleanField.  I have also verified that a field in the LMS that is defined as NullBooleanField is indeed nullable at the database level.  (See certificates_certificategenerationcoursesetting.include_hours_of_effort.)